### PR TITLE
[com_banners] tracks view: searchtools ordering and bug corrections

### PR DESCRIPTION
**Pull request by andrepereiradasilva**
#### Description

This PR corrects bugs in the com_banners tracks views searchtools and matches the searchtools layout with other views.
###### Before PR

image
###### After PR

image
#### How to test
1. Use latest staging
2. Create banners clients, then create some banners and associate with those clients
3. Confirm you have some banner tracks for the tests (see note below)
4. Go to Components -> Banners -> Tracks
5. Check there are several bugs in the searchtools
6. Apply patch
7. Check the search, ordering select box and the filters are working fine.
8. Test desktop, tablet and mobile view (ex: reduce browser width).

Note: If you don't have tracks records you can generate some in the database (`#__banners_tracks` table), example:

``` sql
INSERT INTO `<prefix>_banner_tracks` (`track_date`, `track_type`, `banner_id`, `count`) VALUES ('2016-04-11 00:00:00', '1', '1', '5');
INSERT INTO `<prefix>_banner_tracks` (`track_date`, `track_type`, `banner_id`, `count`) VALUES ('2016-01-11 00:00:00', '2', '1', '7');
INSERT INTO `<prefix>_banner_tracks` (`track_date`, `track_type`, `banner_id`, `count`) VALUES ('2015-04-11 00:00:00', '2', '2', '9');
INSERT INTO `<prefix>_banner_tracks` (`track_date`, `track_type`, `banner_id`, `count`) VALUES ('2016-02-11 00:00:00', '2', '2', '12');
```

### CODE REVIEW

**Comment by franz-wohlkoenig** [comment]
I have tested this item :white_check_mark: successfully on 936080ac4a43be7c5abe4dc1ab3acb13a1aafaeb<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/9951">issues.joomla.org/joomla-cms/9951</a>.</sub>

**Comment by andrepereiradasilva** [comment]
@franz-wohlkoenig just added a better hint (placeholder) in the `begin` and `end` date filters.
IMO, your test remains a success.

File: administrator/language/en-GB/en-GB.com_banners.ini [hunk 1]
```diff
@@ -29,7 +29,9 @@ COM_BANNERS_BATCH_CLIENT_LABEL_DESC="Not making a selection will keep the origin
 COM_BANNERS_BATCH_CLIENT_NOCHANGE="- Keep original Client -"
 COM_BANNERS_BATCH_OPTIONS="Batch process the selected banners"
 COM_BANNERS_BATCH_TIP="If a category is selected for move/copy, any actions selected will be applied to the copied or moved banners. Otherwise, all actions are applied to the selected banners."
-COM_BANNERS_BEGIN_LABEL="Begin Date:"
+COM_BANNERS_BEGIN_DESC="Banner begin date"
+COM_BANNERS_BEGIN_HINT="Begin date (yyyy-mm-dd)"
+COM_BANNERS_BEGIN_LABEL="Begin Date"
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.com_banners.ini]: Please put the : back

**Comment by RonakParmar** [comment]
I have applied this PR and found that "clear" button dose not clear the selected "Begin date" and "End date" filters.
Otherwise PR successfully added search tool as per other Joomla layouts.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/9951">issues.joomla.org/joomla-cms/9951</a>.</sub>

**Comment by andrepereiradasilva** [comment]
@RonakParmar just tried and it clears the dates.
Please, clear browser cache, test again and, if ok, mark as success.

**Comment by RonakParmar** [comment]
@andrepereiradasilva I have deleted my browser cache and also tested in private window still not able to clear selected date using "clear" button.

**Comment by andrepereiradasilva** [comment]
what browser are you using? can you make a small gif video?

**Comment by andrepereiradasilva** [comment]
@RonakParmar is your problem the same as this one? https://github.com/joomla/joomla-cms/issues/7856

**Comment by RonakParmar** [comment]
I think yes, for begin and end date, calendar is used. So may be because of this issue, I am not able to clear the date filters.
@andrepereiradasilva Thanks for the reference issue.

**Comment by RonakParmar** [comment]
I have tested this item :white_check_mark: successfully on 95a92b68a328b846429a4677786229af749c75ad<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/9951">issues.joomla.org/joomla-cms/9951</a>.</sub>

**Comment by andrepereiradasilva** [comment]
@franz-wohlkoenig can you just mark as tested success again so it as 2 success tests and can go RTC?

**Comment by franz-wohlkoenig** [comment]
I have tested this item :white_check_mark: successfully on 95a92b68a328b846429a4677786229af749c75ad<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/9951">issues.joomla.org/joomla-cms/9951</a>.</sub>

**Comment by brianteeman** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/9951">issues.joomla.org/joomla-cms/9951</a>.</sub>

**Comment by andrepereiradasilva** [comment]
@RonakParmar @franz-wohlkoenig thanks for testing

**Pull request merged**